### PR TITLE
i18n(zh-cn): Update pages.mdx

### PR DESCRIPTION
**Pull request by liruifengv**
#### Description

- Update pages.mdx
- Upstream: https://github.com/withastro/starlight/pull/1534

### CODE REVIEW

File: docs/src/content/docs/zh-cn/guides/pages.mdx [hunk 1]
```diff
@@ -106,15 +106,37 @@ import CustomComponent from './CustomComponent.astro';
...
+		{ label: 'Home', href: '/' },
+		{
+			label: 'Constellations',
+			items: [
+				{ label: 'Andromeda', href: '/andromeda/' },
+				{ label: 'Orion', href: '/orion/', isCurrent: true },
+				{ label: 'Ursa Minor', href: '/ursa-minor/', badge: 'Stub' },
+			],
+		},
+	]}
```
Comment (delucis) [inline docs/src/content/docs/zh-cn/guides/pages.mdx:135]: Do you think the example would be easier to understand with translated content @liruifengv? For example, “stub” seems to be “小作品” on Wikipedia ([source](https://zh.wikipedia.org/wiki/Wikipedia:%E5%B0%8F%E4%BD%9C%E5%93%81)) and maybe there are familiar terms for constellation names that could be used?

**Review by delucis — APPROVED** [review]
Looks great @liruifengv! Left a suggestion but up to you if you want to do something with that — I’m happy either way.

File: docs/src/content/docs/zh-cn/guides/pages.mdx (hunk 1 above)
Comment (liruifengv) [reply docs/src/content/docs/zh-cn/guides/pages.mdx:135]: constellation can be translated. Stub I need to take a look. Maybe don't translate it

Comment (delucis) [reply docs/src/content/docs/zh-cn/guides/pages.mdx:135]: You can also pick different constellations if it helps, I picked Andromda, Orion, and Ursa Minor at random. Stub seemed to be translated in the Wikipedia context for a short incomplete article, but happy for you to decide what is clearest!

Comment (liruifengv) [reply docs/src/content/docs/zh-cn/guides/pages.mdx:135]: I'll pick some interesting ones.

**Comment by liruifengv** [comment]
I updated it. I think "Stub" can be left untranslated without affecting the overall reading

**Review by delucis — APPROVED** [review]
Thank you @liruifengv! I’m happy if you are 🌠

**Pull request merged**